assets/js/wpbdp.js

```diff
@@ -275,6 +275,9 @@ WPBDP.fileUpload = {
                         if ( ! t._admin_nonce ) {
                             t._slotsRemaining++;
                             $( '#image-slots-remaining' ).text( t._slotsRemaining );
+							if(t._slotsRemaining < t._slots){
```

Comment:
Let's update this line to match the codestyling in other lines. It should be:
`if ( t._slotsRemaining < t._slots ) {`